### PR TITLE
Use Echo to set up the local disco option

### PR DESCRIPTION
**Pull request by orta**
It has the same structure as the BNMO options - but also forces all admins into using it.

### CODE REVIEW

File: Artsy/App/ArtsyEcho+LocalDisco.m [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+/// To be kept in lock-step with the corresponding echo value, and updated when there is a breaking Maps change.
+/// https://echo-web-production.herokuapp.com/accounts/1/features
+///
+NSInteger const ARLocalDiscoCurrentVersionCompatibility = 0;
+
+@implementation ArtsyEcho (LocalDiscovery)
+
+- (BOOL)shouldShowLocalDiscovery
+{
+    return (self.features[@"AREnableLocalDiscovery"].state && self.isLocalDiscoCompatible) || [ARAppStatus isBetaDevOrAdmin];
```
Comment (ashfurrow) [inline Artsy/App/ArtsyEcho+LocalDisco.m]: Can we add a check for `is iPad?` here? We don't support LD on iPad.

Comment (ashfurrow) [reply Artsy/App/ArtsyEcho+LocalDisco.m]: Or maybe in the other function.

Comment (orta) [reply Artsy/App/ArtsyEcho+LocalDisco.m]: yeah, that makes sense

**Comment by orta** [comment]
OK, that's in and it should green

**Pull request merged**